System/app/Http/Controllers/SupplierController.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use App\Suppliers;
+use App\Person;
+
+class SupplierController extends Controller
+{
+    public function index(Request $request)
+    {
+        if(!$request->ajax()) return redirect('/');
+        
+        $search=$request->search;
+        $criteria=$request->criteria;
+
+        if($search=='')
+        {
+            $Supplier=Suppliers::join('people','suppliers.id','=','people.id')
```

Comment:
Please use lower camel case for variable declarations.
Ex: $users, $bugsBunny
And check if your variable need to be singular or plural